core/src/stages/connect.cpp

```diff
@@ -176,7 +184,7 @@ void Connect::compute(const InterfaceState& from, const InterfaceState& to) {
 	if (!solution)  // success == false or merging failed: store sequentially
 		solution = makeSequential(trajectory_planner_vector, intermediate_scenes, from, to);
 	if (!success)  // error during sequential planning
-		solution->markAsFailure();
+		solution->markAsFailure("Hardcode an error message");
```

Comment:
Can you add a comment to explain this hard coded failure msg?

Reply:
I was just testing something and forgot to remove this.